server/views/partials/viewCases.njk

```diff
@@ -5,12 +5,16 @@
 {% set pageTitle = applicationName + " - view an approved licence hardstop" %}
 {% set pageId = 'view-hardstop-cases-page' %}
 
-{% macro offenderName(offenderName, prisonerNumber, licenceId, isClickable, versionOfId, licenceStatus, index) %}
+{% macro offenderName(offenderName, prisonerNumber, licenceId, isClickable, versionOfId, licenceStatus, index, isHardStop) %}
     <div class="caseload-offender-name">
         {% if isClickable %}
-            {% set link = "/licence/view/id/" + licenceId + "/show" %}
-            {% if versionOfId and licenceStatus === 'SUBMITTED' %}
-                {% set link = link + "?lastApprovedVersion=" + versionOfId %}
+            {% if isHardStop %}
+                {% set link = "/licence/hardstop/create/id/" + licenceId + "/check-your-answers" if licenceId else "/licence/hardstop/create/nomisId/" + prisonerNumber + "/confirm"%}
+            {% else %}
+                {% set link = "/licence/view/id/" + licenceId + "/show" %}
+                {% if versionOfId and licenceStatus === 'SUBMITTED' %}
+                    {% set link = link + "?lastApprovedVersion=" + versionOfId %}
+                {% endif %}
```

Comment:
this is not part of that ticket.. we're looking in to handling this outside of the nunjucks as its quite complicated.

This ticket just covers the create journey flow without the entry point - so the page would exist but you'd have to hit it via the url directly

Reply:
I have removed this code.
We can access licence creation flow only using url for now.